command_handler.py

```diff
@@ -15,17 +15,17 @@ async def handle(self, message: types.Message):
             logger.debug("USER TYPED UNKNOWN COMMAND " + message.text)
 
     async def __handleHelp(self, message):
-        await message.reply(text="This bot is going to manage cpop\.tw\.\n"
+        await message.reply(text="This bot is going to manage cpop\\.tw\\.\n"
```

Comment:
i think another way of fixing it is to use a **multiline string** https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string

but we'll probably change this text anyway, so now worth spending time on formatting the string perfectly

Reply:
Just tried, multiline string has the same escaping issue.
I think `\\` is fine, the backslash escape is for telegram MarkdownV2 specs, and the second backslash is for python to escape the backslash.

Reply:
maybe we should switch back to HTML, use multiline string and format it with HTML to avoid this messy escaping.
I'll try that.